src/views/Simulation/utils/shapes.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+export const PACKAGE_ICON_LINES = [
+  // Package outline
+  [
+    [0, 0.25],
+    [0, 0.75],
+    [0.5, 1],
+    [1, 0.75],
+    [1, 0.25],
+    [0.5, 0],
+    [0, 0.25],
+    [0, 0.75],
+  ],
+  // Package inner lines
+  [
+    [0, 0.25],
+    [0.5, 0.5],
+    [0.5, 1],
+  ],
+  [
+    [0.5, 0.5],
+    [1, 0.25],
+  ],
+  [
+    [0.25, 0.125],
+    [0.75, 0.375],
+  ],
+];
+
+export const GEAR_ICON_LINES = [
+  [
+    [0, 0.25],
+    [0, 0.75],
+    [0.5, 1],
+    [1, 0.75],
+    [1, 0.25],
+    [0.5, 0],
+    [0, 0.25],
+    [0, 0.75],
+  ],
+];
+
+export const FACTORY_ICON_LINES = [
+  // Outline
+  [
+    [0, 0],
+    [0.333, 0],
+    [0.333, 0.5],
+    [0.666, 0.25],
+    [0.666, 0.5],
+    [1, 0.25],
+    [1, 1],
+    [0, 1],
+    [0, 0],
+    [0.333, 0],
+  ],
+  // Dashes
+  [
+    [0.1, 0.8],
+    [0.3, 0.8],
+  ],
+  [
+    [0.4, 0.8],
+    [0.6, 0.8],
+  ],
+  [
+    [0.7, 0.8],
+    [0.9, 0.8],
+  ],
+];
```

Comment:
In the futur, maybe it will be more convenient to use 2D sprite instead of pure vectorial shapes.